src/plugins/dam/components/image-asset-picker-modal/index.jsx

```diff
@@ -29,6 +29,7 @@ class ImageAssetPickerModal extends React.Component {
     galleryNode: PropTypes.instanceOf(Message),
     isOpen: PropTypes.bool,
     label: PropTypes.string,
+    modalTitle: PropTypes.string,
```

Comment:
@bhsiao55 since you're already in the modal, you can just call this title.

Reply:
yup will change